flowzone.yml

```diff
@@ -1486,12 +1489,26 @@ jobs:
           org: ${{ github.repository_owner }}
           default_branch: ${{ github.event.repository.default_branch }}
           url: https://${{ inputs.cloudflare_website }}.pages.dev
+      - name: Decide what branch name to send CF
+        id: cf_branch
+        run: |
+          CF_BRANCH=""
+          echo ${{ fromJSON(needs.event_types.outputs.pr_merged) }}
+          if [[ ${{ fromJSON(needs.event_types.outputs.pr_merged) }} == 'true' ]]
+          then
+            CF_BRANCH=${{ github.event.repository.default_branch }}
+          else
+            CF_BRANCH=${{ github.event.pull_request.head.ref }}
+          fi
+          echo $CF_BRANCH
+          echo "CF_BRANCH=${CF_BRANCH}" >> $GITHUB_ENV
+        shell: bash
       - name: Cloudflare Pages
         uses: cloudflare/wrangler-action@2.0.0
         with:
           apiToken: ${{ secrets.CF_API_TOKEN }}
           accountId: ${{ secrets.CF_ACCOUNT_ID }}
-          command: pages publish --project-name=${{ inputs.cloudflare_website }} build/ >> $GITHUB_STEP_SUMMARY
+          command: pages publish --branch ${{ env.CF_BRANCH }} --project-name=${{ inputs.cloudflare_website }} build/ >> $GITHUB_STEP_SUMMARY
```

Comment:
can you please break this onto multiple lines for readability using (e.g.):

```
command: |
  pages publish \
    --branch ${{ ...
```

Reply:
Probably an issue with how the CF wrangler action handles the `command` input - but breaking it down into multiple lines doesn't work - 

see logs here - https://github.com/balena-io-hardware/meta-hardwareProcess-doc/actions/runs/3576740889/jobs/6015096551#step:9:85

code here - https://github.com/product-os/flowzone/pull/342/files#diff-d899bb0fa13938dbb09e245516c63598abe5837797ab4bdf9860e7f1b70f3cadR1503

Unless there's another way to do this, I guess we are stuck with one big line unfortunately.